app/models/game.rb

```diff
@@ -254,12 +210,12 @@ def save_state state
     self.state = state.to_yaml
   end
 
-  def get_highest_card cards, first_suit_played, trump, start_index
+  def get_highest_card cards, first_suit_played, trump
     return cards.first if cards.size <= 1
 
     # if trump was played, ignore other cards
     # an ace indicates no trump, so ignore it if that's the case
-    trump_cards = (trump.suit =~ /ace/i) ? [] : cards.select { |c| c.suit == trump.suit }
+    trump_cards = (trump.value == 12) ? [] : cards.select { |c| c.suit == trump.suit }
```

Comment:
Oops haha yea. Whatever your implementation is fine with me. Fuck string
comparisons
On May 15, 2015 9:14 AM, "Greg Wilbur" <notifications@github.com> wrote:

> In app/models/game.rb
> <https://github.com/bcwik9/crushcard/pull/17#discussion_r30406615>:
>
> >      return cards.first if cards.size <= 1
> >
> >      # if trump was played, ignore other cards
> >      # an ace indicates no trump, so ignore it if that's the case
> > -    trump_cards = (trump.suit =~ /ace/i) ? [] : cards.select { |c| c.suit == trump.suit }
> > +    trump_cards = (trump.value == 12) ? [] : cards.select { |c| c.suit == trump.suit }
>
> did you mean to use value_name instead of suit then?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bcwik9/crushcard/pull/17/files#r30406615>.
>